app/components/c-header-links.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div :class="b()">
+    <div v-if="userLoggedIn" :class="b('item', { active: accountActive })">
+      <a :class="b('link')" :href="$t('c-header-links.linkAccount')">
+        <e-icon
+          :class="b('icon')"
+          inline
+          icon="i-account"
+          width="25px"
+          height="25px"
+        />
+        <span :class="b('label')">{{ username }}</span>
+      </a>
+    </div>
+    <div v-if="userLoggedIn" :class="b('item', { active: wishlistActive })">
+      <a :class="b('link')" :href="$t('c-header-links.linkWishlist')">
+        <e-icon
+          :class="b('icon')"
+          inline
+          icon="i-wishlist"
+          width="25px"
+          height="25px"
+        />
+        <span :class="b('label')">{{ $t('c-header-links.labelWishlist') }}</span>
+      </a>
+    </div>
+    <div v-if="!userLoggedIn" :class="b('item', { active: loginActive })">
+      <a :class="b('link')" :href="$t('c-header-links.linkLogin')">
+        <e-icon
+          :class="b('icon')"
+          inline
+          icon="i-account"
+          width="25px"
+          height="25px"
+        />
+        <span :class="b('label')">{{ $t('c-header-links.labelLogin') }}</span>
+      </a>
+    </div>
+  </div>
+</template>
+
+<script>
+  import eIcon from '@/components/e-icon';
+
+  export default {
+    name: 'c-header-links',
+    components: {
+      eIcon,
+    },
+    // mixins: [],
+
+    props: {
+      isAccountActive: {
+        type: Boolean,
+        default: false,
+      },
+      isWishlistActive: {
+        type: Boolean,
+        default: false,
+      },
+      isLoginActive: {
```

Comment:
[ ] Rename to login, "is" Prefix is usually associated with getters
[ ] Add documentation